letsencrypt/tests/cli_test.py

```diff
@@ -235,7 +235,7 @@ def test_configurator_selection(self, mock_exe_exists):
         with mock.patch("letsencrypt.cli._init_le_client") as mock_init:
             with mock.patch("letsencrypt.cli._auth_from_domains"):
                 self._call(["certonly", "--manual", "-d", "foo.bar"])
-                auth = mock_init.call_args[0][2]
+                _config, auth, _installer = mock_init.call_args[0]
```

Comment:
nit: The names of the two unused variables should the idiomatic `_` or prefixed with `unused`.